bot.py

```diff
@@ -0,0 +1,34 @@
+import discord
```

Comment:
Jeśli to ma być plik, który odpalamy w terminalu to brakuje tutaj shebanga https://pktiuk.github.io/notatki_z_jezykow/Python%F0%9F%90%8D/1_ogolne_notatki/